python/semantic_kernel/template_engine/blocks/code_block.py

```diff
@@ -96,7 +96,7 @@ async def _render_function_call_async(self, f_block: FunctionIdBlock, context):
             input_value = self._tokens[1].render(variables_clone)
             variables_clone.update(input_value)
 
-        result = await function.invoke_with_custom_input_async(
+        result = await function.invoke_with_vars_async(
```

Comment:
invoke_with_vars_async() does not support the context.skills argument right now

Reply:
I think this is better, showing how to get to the point without too many ceremonials, which is one of the feedback we've received.